src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 10px;
+  gap: $gap;
+  width: ($box-size * 7) + ($gap * 6) + 20px;
+
+  @for $lastDay from 28 through 31 {
+    &__month-length--#{$lastDay}
+    &__day:nth-child(n + #{$lastDay} + 1) {
+      display: none;
```

Comment:
You have to correct this loop, it doesn`t work. The number of cells don`t change when you change number of days in your class.

Reply:
Fixed, thx